src/Opserver.Core/Data/Exceptions/ExceptionStore.cs

```diff
@@ -171,6 +172,24 @@ public class SearchParams
             public Guid? StartAt { get; set; }
             public ExceptionSorts Sort { get; set; }
             public Guid? Id { get; set; }
+            public string Url { get; set; }
+            public string Host { get; set; }
+
+            public bool IsNonDefault
+            {
+                get
+                {
+                    if (Host.HasValue()) return true;
+                    if (Url.HasValue()) return true;
+                    if (StartDate != null) return true;
+                    if (EndDate != null) return true;
+                    return false;
+                }
+            }
```

Comment:
Doh, one comment didn't submit - check me here but cheaper/slimmer:
```suggestion
            public bool IsNonDefault =>
              Host.HasValue() || Url.HasValue() || StartDate.HasValue || EndDate.HasValue;
```

Reply:
(also suggest renaming this to "HasFilterOptions" or something since the current name isn't quite accurate to intent)